geonode/qgis_server/signals.py

```diff
@@ -236,6 +236,26 @@ def qgis_server_post_save(instance, sender, **kwargs):
             )
         )
 
+    # QLR link layer workspace
+    ogc_qlr_url = urljoin(
+        base_url,
+        reverse(
+            'qgis_server:download-qlr',
+            kwargs={'layername': instance.name}))
+    logger.debug('qlr_download_url: %s' % ogc_qlr_url)
+    link_name = 'QGIS Layer file'
+    link_mime = 'QLR'
```

Comment:
QLR is XML, so it should be `application/xml` ?

Reply:
I'm not sure about this. 
`link_mime` is `zip` in zip link.
i've just updated to `application/xml`, but it has no effect.
do you think we should use `application/xml` instead?

Reply:
I don't think you will see any effect in Geonode. But it's better to provide the good MIME in the DB, for catalogue API such as CSW.

ZIP looks wrong, it should be `application/zip`. Read https://en.wikipedia.org/wiki/Media_type and look for `zip`